src/sage/rings/polynomial/multi_polynomial_ring_base.pyx

```diff
@@ -432,8 +432,9 @@ cdef class MPolynomialRing_base(CommutativeRing):
             Also, if the solution is not unique, it spits out one solution,
             without any notice that there are more.
 
-            Lastly, the interpolation function for univariate polynomial rings
-            is called :meth:`lagrange_polynomial`.
+            Lastly, the interpolation function for univariate polynomial rings,
+            :meth:`~sage.rings.polynomial.polynomial_ring.PolynomialRing_field.lagrange_polynomial`,
+            is called.
```

Comment:
This doesn't look correct: this method doesn't call `lagrange_polynomial` at all.

I used `git blame` to find the original commit, https://github.com/sagemath/sage/commit/7eea8892b6d584d74f80c0554ae31a67e58cdff8 and then the issue #34651.  In fact, there is a comment explaining what the intention was: https://github.com/sagemath/sage/issues/34651#issuecomment-1418189041 

So, the author just wanted to notify users that there is a faster variant for the univariate case.  I therefore suggest:

```suggestion
            For interpolation in the univariate case use
            :meth:`~sage.rings.polynomial.polynomial_ring.PolynomialRing_field.lagrange_polynomial`.
```

Reply:
Huh, I didn't actually fact-check it, just come across the suboptimal formatting. Thanks.